Resources/Prototypes/_SpaceCats/Entities/Structures/Machines/Computers/book_printer.yml

```
@@ -0,0 +1,119 @@
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:ChangeConstructionNodeBehavior
+        node: machineFrame
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: ItemSlots
+    slots:
+       bookSlot:
+        whitelistFailPopup: book-printer-component-cannot-put-entity-message
```

yamllint (1.35.1)

[warning] wrong indentation: expected 9 but found 8

(indentation)